fileupload/client_test.go

```diff
@@ -56,6 +56,46 @@ func TestFileUploadNewThenGet(t *testing.T) {
 	}
 }
 
+func TestInterfaceFileUploadNewThenGet(t *testing.T) {
```

Comment:
Could you rename this to `TestFileUploadNewThenGet` and then rename the current test to be `TestFileUploadNewThenGetWithFile`? I think it'll help convey that the former is the newer preferred option.